doc/api/next_api_changes/2018-09-18-DS.rst

```diff
@@ -0,0 +1,32 @@
+Removal of deprecated :mod:`matplotlib.mlab` code
+-------------------------------------------------
+
+Lots of code inside the :mod:`matplotlib.mlab` module which was deprecated
+in Matplotlib 2.2 has been removed. See below for a list:
+
+- `mlab.logspace` (use numpy.logspace instead)
```

Comment:
The former should be inline-code formatting (since there's nothing to link to now) and the replacement should be a link (since we should have intersphinx for these.)

Reply:
I'll fix this all in one go at the end